pom.xml

```
@@ -8,7 +8,7 @@
     <groupId>com.ibm.eventautomation.demos</groupId>
     <artifactId>kafka-connect-loosehangerjeans-source</artifactId>
     <description>Kafka Connect source connector used for generating simulated events for demos and tests</description>
-    <version>0.2.1</version>
+    <version>0.2.2</version>
```

```suggestion
    <version>0.3.0</version>
```

As you're adding a new topic, I think a minor version number change would be more appropriate.

---

ok, done